ls1intum/Artemis
# [Exam] Add exam to course overview

### Checklist

### Motivation and Context


Adds exam banner to course overview in order to simplify navigation.

### Description

- Added banner to html file
- Extracted exams from courses

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Create at least 2 exams with start date in the future
4. Go to course overview
5. Exam banner should be displayed with the data of the exam with the first `startDate` and `endDate` is after now 
(when`endDate` is before now then this exam should not be displayed anymore) 
6. When clicking on the exam banner, you are navigated to the exam

### Screenshots


<img width="1253" alt="image" src="https://user-images.githubusercontent.com/41108487/86141223-15464480-baf2-11ea-80d3-65936e6d2442.png">
